infrastructure.md

```diff
@@ -8,4 +8,100 @@ The best description of the full infrastructure is currently given as an [Ansibl
 
 The plans for the final set-up are given in [this presentation Jonathan prepared for Harmony 2018](https://github.com/ModellingWebLab/WLDocs/blob/master/doc/WL2%20technical%20detail.pdf)
 
+## Getting started
+
+To get started practically, jump down to the deployment section **TODO: ADD LINK**
+
+## Overview
+
+This diagram describes the general purpose Web Lab infrastructure.
+
+![schematic overview](img/infra-1.png)
+
+## Django website
+
+The website is written in Python using [Django](https://docs.djangoproject.com/en/2.2/).
+The Django server serves up a HTML and javascript front-end, which can interact with the Django server backend via a REST API.
+**JONATHAN: DID WE ADD ANYTHING HERE, OR IS THIS ALL HANDLED BY DJANGO?**
+
+The current Django application is slightly cardiac-specific, and is hosted in the [WebLab](https://github.com/ModellingWebLab/WebLab) repository.
+
+The django back-end can access data from various sources (databases, local files, git repositories, a metadata triple store), and can communicate with the "task queue" to execute experiments via a REST API.
+**JONATHAN: IS THIS API DOCUMENTED SOMEWHERE?**
```

Comment:
Yep, https://github.com/ModellingWebLab/WebLab/blob/master/docs/experiment_runner_rest_api.md

Reply:
Eventually we should have it auto-generated with swagger - pending improvements to fc-runner to use a proper REST framework rather than hand-coded old-fashioned CGI.